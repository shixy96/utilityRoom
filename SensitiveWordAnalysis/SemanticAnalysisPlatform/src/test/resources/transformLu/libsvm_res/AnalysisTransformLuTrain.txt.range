x
-1 1
1 0 9
2 0 13
3 0 27
4 0 15
5 0 20.53125
6 0 16.75
7 0 30.5625
8 0 196.875
9 0 34.875
10 0 18.5
11 0 27.75
12 0 18.5
13 0 15.75
14 0 16
15 0 16.5
16 0 16
17 0 17.25
18 0 15
19 0 17.25
20 0 3
21 0 3
