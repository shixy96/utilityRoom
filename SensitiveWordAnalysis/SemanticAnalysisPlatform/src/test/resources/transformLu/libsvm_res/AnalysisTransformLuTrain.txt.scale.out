log2c=5 log2g=-7 rate=82.6517
log2c=-1 log2g=-7 rate=82.9069
log2c=5 log2g=-1 rate=85.4653
log2c=-1 log2g=-1 rate=83.1933
log2c=11 log2g=-7 rate=83.517
log2c=11 log2g=-1 rate=87.0153
log2c=5 log2g=-13 rate=82.9069
log2c=-1 log2g=-13 rate=74.5969
log2c=11 log2g=-13 rate=82.6766
log2c=-3 log2g=-7 rate=74.4911
log2c=-3 log2g=-1 rate=82.9879
log2c=-3 log2g=-13 rate=74.5969
log2c=5 log2g=1 rate=87.3576
log2c=-1 log2g=1 rate=85.7143
log2c=11 log2g=1 rate=87.4323
log2c=-3 log2g=1 rate=84.8179
log2c=9 log2g=-7 rate=82.8011
log2c=9 log2g=-1 rate=86.8908
log2c=9 log2g=-13 rate=82.6829
log2c=9 log2g=1 rate=87.2456
log2c=5 log2g=-11 rate=82.6517
log2c=-1 log2g=-11 rate=74.5969
log2c=11 log2g=-11 rate=82.5957
log2c=-3 log2g=-11 rate=74.5969
log2c=9 log2g=-11 rate=82.6019
log2c=3 log2g=-7 rate=82.6829
log2c=3 log2g=-1 rate=84.93
log2c=3 log2g=-13 rate=74.4787
log2c=3 log2g=1 rate=87.1771
log2c=3 log2g=-11 rate=82.9132
log2c=5 log2g=-5 rate=82.8073
log2c=-1 log2g=-5 rate=82.6642
log2c=11 log2g=-5 rate=83.9153
log2c=-3 log2g=-5 rate=82.9256
log2c=9 log2g=-5 rate=83.9153
log2c=3 log2g=-5 rate=82.7575
log2c=15 log2g=-7 rate=84.4258
log2c=15 log2g=-1 rate=87.3203
log2c=15 log2g=-13 rate=82.6642
log2c=15 log2g=1 rate=86.9219
log2c=15 log2g=-11 rate=82.8011
log2c=15 log2g=-5 rate=85.1976
log2c=5 log2g=-15 rate=74.4787
log2c=-1 log2g=-15 rate=74.5969
log2c=11 log2g=-15 rate=82.577
log2c=-3 log2g=-15 rate=74.5969
log2c=9 log2g=-15 rate=82.6517
log2c=3 log2g=-15 rate=74.5969
log2c=15 log2g=-15 rate=82.826
log2c=-5 log2g=-7 rate=74.5969
log2c=-5 log2g=-1 rate=82.8571
log2c=-5 log2g=-13 rate=74.5969
log2c=-5 log2g=1 rate=82.8447
log2c=-5 log2g=-11 rate=74.5969
log2c=-5 log2g=-5 rate=74.5036
log2c=-5 log2g=-15 rate=74.5969
log2c=5 log2g=3 rate=87.9739
log2c=-1 log2g=3 rate=87.0401
log2c=11 log2g=3 rate=87.395
log2c=-3 log2g=3 rate=85.5649
log2c=9 log2g=3 rate=87.6439
log2c=3 log2g=3 rate=88.0984
log2c=15 log2g=3 rate=86.9468
log2c=-5 log2g=3 rate=84.2266
log2c=7 log2g=-7 rate=82.7202
log2c=7 log2g=-1 rate=86.15
log2c=7 log2g=-13 rate=82.6455
log2c=7 log2g=1 rate=87.6004
log2c=7 log2g=-11 rate=82.6642
log2c=7 log2g=-5 rate=83.4921
log2c=7 log2g=-15 rate=82.9132
log2c=7 log2g=3 rate=87.8494
log2c=5 log2g=-9 rate=82.6829
log2c=-1 log2g=-9 rate=74.4787
log2c=11 log2g=-9 rate=82.7326
log2c=-3 log2g=-9 rate=74.5969
log2c=9 log2g=-9 rate=82.6331
log2c=3 log2g=-9 rate=82.658
log2c=15 log2g=-9 rate=83.7846
log2c=-5 log2g=-9 rate=74.5969
log2c=7 log2g=-9 rate=82.6455
log2c=1 log2g=-7 rate=82.6766
log2c=1 log2g=-1 rate=84.1955
log2c=1 log2g=-13 rate=74.5969
log2c=1 log2g=1 rate=86.2372
log2c=1 log2g=-11 rate=74.4787
log2c=1 log2g=-5 rate=82.6704
log2c=1 log2g=-15 rate=74.5969
log2c=1 log2g=3 rate=87.9676
log2c=1 log2g=-9 rate=82.9132
log2c=5 log2g=-3 rate=83.629
log2c=-1 log2g=-3 rate=82.77
log2c=11 log2g=-3 rate=85.179
log2c=-3 log2g=-3 rate=82.7451
log2c=9 log2g=-3 rate=84.8553
log2c=3 log2g=-3 rate=83.0626
log2c=15 log2g=-3 rate=86.4799
log2c=-5 log2g=-3 rate=82.9069
log2c=7 log2g=-3 rate=84.7121
log2c=1 log2g=-3 rate=82.8634
log2c=13 log2g=-7 rate=83.9527
log2c=13 log2g=-1 rate=87.1335
log2c=13 log2g=-13 rate=82.7015
log2c=13 log2g=1 rate=87.258
log2c=13 log2g=-11 rate=82.7389
log2c=13 log2g=-5 rate=84.5938
log2c=13 log2g=-15 rate=82.4712
log2c=13 log2g=3 rate=87.0339
log2c=13 log2g=-9 rate=82.9879
log2c=13 log2g=-3 rate=85.7703
